zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2717,6 +2754,42 @@ components:
           description: All the variables visible in the root scope.
           type: object
 
+    VariableRequest:
+      type: object
+      properties:
+        variables:
+          description: JSON object representing the variables to set in the element’s scope.
+          additionalProperties: true
+          type: object
+        local:
+          description: |
+            If set to true, the variables will be merged strictly into the local scope (as indicated by elementInstanceKey).
+            Otherwise, the variables are propagated to upper scopes and set at the outermost one.
+
+            Let’s consider the following example:
+
+            There are two scopes '1' and '2'.
+            Scope ‘1’ is the parent scope of ‘2’. The effective variables of the scopes are:
+            1 => { "foo" : 2 }
+            2 => { "bar" : 1 }
+
+            An update request with elementInstanceKey as ‘2’, variables { "foo" : 5 }, and local set
+            to true leaves scope ‘1’ unchanged and adjusts scope ‘2 ‘ to { "bar" : 1, "foo" 5 }.
+
+            By default, with local set to false, scope ‘1’ will be { "foo": 5 }
+            and scope ‘2’ will be { "bar" : 1 }.
```

Comment:
🔧 Some of the `'` are off, I think they are different characters. Probably an issue with the original text and copy-pasting them into the OpenAPI spec.